main.py

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env python3
-
+import time
```

Comment:
Why `import time`?

Reply:
It was used to count the time program runs, and I forgot to remove it.